src/languages/es-ES.ts

```diff
@@ -280,6 +280,28 @@ export default class extends Language {
 		ARGUMENT_RANGE_MAX: (name: string, maximum: number) => `El argumento ${name} acepta un rango de máximo ${maximum} ${maximum === 1 ? 'número' : 'números'}`,
 
 		COMMAND_ADD_DESCRIPTION: `Añade una canción a la cola.`,
+		COMMAND_ADD_EXTENDED: builder.display('add', {
+			extendedHelp: [
+				`¡Agregue canciones a la lista de reproducción y prepárese para el disfrute musical!
+					Puedo jugar desde YouTube, Bandcamp, SoundCloud, Twitch, Vimeo o Mixer.`,
+				'- Para jugar desde YouTube, dame algo para buscar, un enlace de video o un enlace de lista de reproducción.',
+				'- Para jugar desde SoundCloud, dame un enlace de SoundCloud, o si quieres que busque, incluye `--sc` o` --soundcloud` en tu mensaje.',
+				'- Para jugar desde Mixer, dame la URL de un Streamer de Mixer, lo siento pero no puedo (todavía) reproducir VOD de Mixer.',
+				'- Para jugar desde Bandcamp, Twitch o Vimeo, solo dame una URL a un video o lista de reproducción en esas fuentes.'
```

Comment:
Not even going to lie but this feels more Latin American than Spanish.

Also Spanish is a very contextual language, so some stuff is redundant.
```suggestion
					Puedo reproducir desde YouTube, Bandcamp, SoundCloud, Twitch, Vimeo o Mixer.`,
				'- Para reproducir desde YouTube, dame algo para buscar, un enlace de video o un enlace de lista de reproducción.',
				'- Para reproducir desde SoundCloud, dame un enlace de SoundCloud, o si quieres que busque, incluya `--sc` o` --soundcloud` en su mensaje.',
				'- Para reproducir desde Mixer, escribe la URL de un streamer de Mixer, lo siento pero no puedo (todavía) reproducir VODs.',
				'- Para reproducir desde Bandcamp, Twitch o Vimeo, solo necesita escribir un enlace a un video o lista de reproducción.'
```